mediawiki/mediawikipage.py

```diff
@@ -432,11 +476,13 @@ def section(self, section_title):
                 Side effect is to also pull the content which can be slow
             Note:
                 This is a parsing operation and not part of the standard API'''
-        section = '== {0} =='.format(section_title)
+
+        section_regexp = '\n==* {} ==*\n'.format(section_title)
```

Comment:
should this be a '+' not a '*'? It should be at least two '=' chars, right?

Reply:
I copied this from the other place in self.content. but you're right. I'll check what it needs to be.

Reply:
the original author used `*`. While I can't find any place with a single `=` but I'm inclined to trust the author and keep it this. But not very strongly. Thoughts?

Reply:
sure, that's fine